.github/workflows/auto-update.yml

```
@@ -3,36 +3,41 @@ name: Auto update
 env:
   BUILDER_TOOLS_URL: https://github.com/flatpak/flatpak-builder-tools.git
   BUILDER_TOOLS_BRANCH: master
+  TOOLS_DIR: builder-tools
 
 on:
-  schedule:
-    - cron: '45 6 * * *'
+  pull_request: {}
   workflow_dispatch: {}
 
 jobs:
   update:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
     steps:
       - name: Install dependencies
         run: |
           set -e
-          sudo apt install python3-toml python3-aiohttp
+          sudo apt install \
+            python3-aiohttp \
+            python3-toml
 
       - uses: actions/checkout@v2
```

Let's try copying clone job from Code-OSS, maybe the last `git push` will stop failing then.
```suggestion
      - name: Checkout repository
        uses: actions/checkout@v3
        with:
          ref: ${{ github.event.pull_request.head.ref }}
          submodules: true
```